java/refactoring.java/src/org/netbeans/modules/refactoring/java/plugins/PushDownTransformer.java

```diff
@@ -62,11 +67,18 @@ public Problem getProblem() {
         return problem;
     }
 
-    public PushDownTransformer(MemberInfo<ElementHandle<? extends Element>> members[]) {
+    public PushDownTransformer(FileObject originFile, MemberInfo<ElementHandle<? extends Element>> members[]) {
+        this.originFile = originFile;
         this.members = members;
         this.translateQueue = new LinkedList<>();
     }
 
+    @Override
+    public void setWorkingCopy(WorkingCopy workingCopy) throws ToPhaseException {
+        SourceUtils.forceSource(workingCopy, originFile);
+        super.setWorkingCopy(workingCopy); //To change body of generated methods, choose Tools | Templates.
```

Comment:
Delete the comment.

Reply:
Right. Sorry, I mistakenly based this branch on a different branch, fixed. There should only be one file changed, the ClassPathProvider.